src/test/scala/zio/akka/cluster/sharding/ShardingSpec.scala

```diff
@@ -183,8 +191,8 @@ object ShardingSpec extends DefaultRunnableSpec {
                 p2        <- Promise.make[Nothing, Unit]
                 onMessage1 = (_: String) => p1.succeed(()).unit
                 onMessage2 = (_: String) => p2.succeed(()).unit
-                sharding1 <- Sharding.start(shardName, onMessage1).provideLayer(ZLayer.succeedMany(a1))
-                _         <- Sharding.start(shardName, onMessage2).provideLayer(ZLayer.succeedMany(a2))
+                sharding1 <- Sharding.start(shardName, onMessage1).provideLayer(ZLayer.succeedEnvironment(a1))
```

Comment:
Can we simply use `succeed`?

Reply:
not sure what you mean by succeed,
but I changed it to
```
Sharding.start(shardName, onMessage1).provideEnvironment(a1)
```